hq/app/AppComponents.scala

```diff
@@ -8,12 +12,15 @@ import play.api.mvc.{AnyContent, BodyParser, ControllerComponents}
 import play.api.routing.Router
 import play.filters.csrf.CSRFComponents
 import router.Routes
+import service._
 
 
 class AppComponents(context: Context)
   extends BuiltInComponentsFromContext(context)
   with CSRFComponents
-  with AhcWSComponents with AssetsComponents {
+  with AhcWSComponents with AssetsComponents  {
+
+  val cacheService = new CacheService(configuration)
```

Comment:
What are the benefits and risks of having many cache services vs. one? I have a (maybe unfounded) preference of splitting the responsibilities into two services:
```
val sgsUpdater = new SecurityGroupsUpdater(configuration)
val credUpdater = new CredentialServiceUpdater(configuration)
```
Would this have any impact on performance? Would it make it harder or easier to expand functionality or debug?

Reply:
@katebee performance wise, I don't expect any difference at all really. Can be separated if you like.